CIScripts/Test/Tests/vRouterAgent.Tests.ps1

```diff
@@ -0,0 +1,131 @@
+Param (
```

Comment:
Why this file is named `vRouterAgent.Tests`? Shouldn't it be `AgentService.Tests`? In current test we have `vRouterAgentTest.ps1` which tests vRouter-Agent integration, so this name will be confusing.

Reply:
`vRouterAgentTest.ps1` will probably be renamed in the future.

 > Shouldn't it be `AgentService.Tests`?
Not sure, since we test MSI as well.

Reply:
I'd name it just `Agent.Tests`, so it won't get confused with current `vRouterAgentTest.ps1`.